### PR TITLE
Front end branch routing

### PR DESCRIPTION
**Pull request by ROlearyPro**
Lots of front end updates, including modals and custom fonts!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,8 +6,12 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.3.3",
+    "cypress": "^13.15.1",
     "react": "^18.3.1",
+    "react-bootstrap": "^2.10.5",
     "react-dom": "^18.3.1",
+    "react-modal": "^3.16.1",
```
Comment (jchirch) [inline package.json:14]: Simple, powerful piece of tech. I'll definitely reuse this

File: src/App/App.css [hunk 2]
```diff
@@ -1,9 +1,12 @@
+@font-face {
```
Comment (jchirch) [inline src/App/App.css:1]: Perfect font choice, way to contribute to this thematically heavy project!

File: src/MainPage/MainPage.css [hunk 3]
```diff
@@ -102,66 +144,148 @@
...
+  align-items: center;
+  /* outline: 5px ridge blue; */
   border-radius: 50px;
-  border: 5px, solid blue;
+  border: 5px, ridge rgb(0, 145, 255);
 }
 
-.App-link {
-  color: #61dafb;
+.train-button {
```
Comment (jchirch) [inline src/MainPage/MainPage.css:221]: CSS looks good, we need to clean up comments before pushing to production.

File: src/MainPage/MainPage.js [hunk 4]
```diff
@@ -42,13 +61,66 @@ function MainPage() {
...
+                  <Button variant="secondary" onClick={handleHelpVisible}>
+                    Close
+                  </Button>
+                </Modal.Footer>
+              </Modal>
+            </button>
+          </div>
+          <div className='button-containers-2'>
+
+            <button type="button" className='train-button'>
```
Comment (jchirch) [inline src/MainPage/MainPage.js:88]: Excellent structure, I appreciate the attention to detail with semantic HTML for keeping project accessible.

File: src/PartyMenu/PartyMenu.css [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  min-height: 100vh;
+  max-height: 100vh;
+  /* min-width: 100vmax; */
+  max-width: 100vw;
+  color: white;  align-items: center;
+  align-self: center;
+
+  background-size: contain;
+
+  a:link {
```
Comment (jchirch) [inline src/PartyMenu/PartyMenu.css:16]: For developer empath, it may be good to rename "a" to something descriptive. Unsure what this does at a glance.

**Comment by MDelarosa1993** [comment]
Great job on these updates! I really appreciate the consistency in the font choice across the application—it adds a cohesive feel that enhances the overall user experience. The attention to detail in the CSS adjustments, especially with the background images and layout changes, shows a strong understanding of responsive design principles. I'm excited to see how these improvements contribute to the project's thematic depth. Keep up the fantastic work!

**Pull request merged**